test/lib/usd/translators/testUsdExportLayerAttributes.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
```

Comment:
It's mostly a vestige from the core USD repo at this point, but the rest of the tests here use this shebang line:

```Python
#!/pxrpythonsubst
```

Just for consistency, I'd suggest using that. That way if/when we do any kind of shebang replacement, we'll catch this test too.

Reply:
Done!